com.unity.netcode.gameobjects/Runtime/Core/NetworkObject.cs

```diff
@@ -429,8 +425,7 @@ internal void SnapshotDespawn()
         internal void SnapshotDespawn(ulong clientId)
         {
             var command = GetDespawnCommand();
-            command.TargetClientIds = new List<ulong>();
-            command.TargetClientIds.Add(clientId);
+            command.TargetClientIds = new List<ulong> { clientId };
```

Comment:
Ok, yeah, those are great changes, but they make the PR harder to process and possibly harder to merge later. We should try to have PRs that address just one thing and side clean-up PRs are fine too :-). </nitpick> </jeffshouldgethiscoffee>